apps/hosts/serializers.py

```diff
@@ -42,34 +42,6 @@ class Meta:
         fields = ('id', 'user', 'team_name', 'status', 'permissions',)
 
 
-class InviteHostToTeamSerializer(serializers.Serializer):
```

Comment:
I am not sure why this part of code is removed. Am I missing something here?

Reply:
Initially, the E-mail and Id would be in the request, but here, we're decoding the data from the URL. So the serializer needed a major overhaul and I found this method to be much efficient than re-writing the serializer.